README.md

```diff
@@ -4,120 +4,66 @@
 [![coveralls](https://img.shields.io/coveralls/pfnet/chainer.svg)](https://coveralls.io/github/pfnet/chainer)
 [![Read the Docs](https://readthedocs.org/projects/chainer/badge/?version=stable)](http://docs.chainer.org/en/stable/?badge=stable)
 
-# Chainer: a neural network framework
-
-## Requirements
-
-Chainer is tested on Ubuntu 14.04 and CentOS 7. We recommend them to use Chainer, though it may run on other systems as well.
-
-Minimum requirements:
-- Python 2.7.6+, 3.4.3+, 3.5.1+, 3.6.0+
-- NumPy 1.9, 1.10, 1.11, 1.12
-- Six 1.9
-
-Requirements for some features:
-- CUDA support
-  - CUDA 6.5, 7.0, 7.5, 8.0
-  - filelock
-  - g++ 4.8.4+
-- cuDNN support
-  - cuDNN v2, v3, v4, v5, v5.1, v6
-- Caffe model support
-  - Protocol Buffers (pip install protobuf)
-    - protobuf>=3.0.0 is required for Py3
-- Image dataset support
-  - Pillow
-- HDF5 serialization support
-  - h5py 2.5.0
-- Testing utilities
-  - Mock
-  - Nose
+# Chainer: a deep learning framework
 
-## Installation
+[Website](http://chainer.org/)
+| [Docs](http://docs.chainer.org/en/stable/)
+| [Install Guide](http://docs.chainer.org/en/stable/install.html)
+| [Tutorial](http://docs.chainer.org/en/stable/tutorial/)
+| Examples ([Official](https://github.com/pfnet/chainer/blob/master/examples), [External](https://github.com/pfnet/chainer/wiki/External-examples))
+| Forum ([en](https://groups.google.com/forum/#!forum/chainer), [ja](https://groups.google.com/forum/#!forum/chainer-jp))
+| Slack ([en](https://bit.ly/chainer-slack), [ja](https://bit.ly/chainer-jp-slack))
+| Twitter ([en](https://twitter.com/ChainerOfficial), [ja](https://twitter.com/ChainerJP))
 
```

Comment:
How about making these links bold style? It could be hard for some people to notice.